typing/includemod.mli

```diff
@@ -154,15 +162,15 @@ val is_runtime_component: Types.signature_item -> bool
 (* Typechecking *)
 
 val modtypes:
-  loc:Location.t -> Env.t -> mark:mark ->
+  loc:Location.t -> Env.t -> mark:mark -> modes:unit option ->
```

Comment:
In addition to changing these `modes` parameters to carry the modes, I think it might be worth making a specific type like:
```
type inclusion_mode =
  | All
  | Specific of { from; to_ }
```

Reply:
yeah that's what I did in the last commit. Although, the `Specific` constructor still doesn't carry modes.